environment.yaml

```diff
@@ -21,5 +21,6 @@ dependencies:
 - bottleneck=1.3.2
 - zarr=2.8.3
 - pip:
+  - pooch 
```

Comment:
Eh. What do we need `pooch` for?

Reply:
We need `pooch` to be able to use the air temperature dataset from xarray's tutorials module (otherwise tests fail)

Reply:
Hmm I see what you're going for. I'd rather avoid making a test depend on external data as a general best practice. In addition, this is a single test case, requiring a new single-use dependency. I don't think xarray even uses air_temp data in their tests, it's only included in doc build environments for demos.

I understand you're cranking this out before break but it's something I would sacrifice blood to avoid. Might be something I could replace while you're out on vacation?

Reply:
I kinda really like using this dataset for testing though. It is a single test case, but one that is very essential to our pipeline. Why are you against it out of curiosity?

Reply:
@dgergel Unit testing with external data is usually a crutch for poor testing practices. With a unit test you want to only test some behavior in the code, not the code + external data. 

Like to keep package dependencies as light as possible, and test fast. Here we've got a new package dependency, and IO to some external data... this is added weight and more things to maintain.

Reply:
Got it. This makes sense. I've updated it to use some fake test data that we produce on the fly while testing. Marking resolved.